lib/ace/mode/lua/luaparse.js

```diff
@@ -1935,6 +1935,7 @@ define(function(require, exports, module) {
     if (!_options) _options = {};
 
     input = _input || '';
+    input = input.replace(/\r\n/g, '\n');
```

Comment:
this file is autogenerated from https://github.com/oxyc/luaparse, so the fix will be lost on the next update.
It is better to change https://github.com/ajaxorg/ace/blob/master/lib/ace/mode/lua_worker.js#L48 to `this.doc.getAllLines().join("\n")`